README.md

```diff
@@ -10,16 +10,16 @@ For the first time, designers can create **and ship** beautiful animations witho
 ## Installing
 
 ### iOS and Android
-Install `lottie-react-native` (latest) and `lottie-ios` (3.4.0):
+Install `lottie-react-native` (latest):
 
 ```
-yarn add lottie-react-native lottie-ios@3.4.0
```

Comment:
This may lead to future issues as new versions of `lottie-ios` are released. Having it as a peer dependency creates certainty of what's released works as expected.

Reply:
But we are forcing it in our podspec and it will work with our compatible version.

Reply:
That's fine for new installations but upgrades will end up with lots of versioning issues.

Reply:
Wait, since we are now aiming for a major version release we might include this one safely and add a migration document.

Reply:
Yes, we need to include it in v6.0.0 document.